pkg/controllers/config/labels.go

```diff
@@ -16,17 +16,28 @@ package config
 
 const (
 	// Well known labels on ExternalEntities so that they can be selected by Antrea NetworkPolicies.
-	ExternalEntityLabelKeyPostfix    = "nephe"
-	ExternalEntityLabelKeyNamespace  = "namespace." + ExternalEntityLabelKeyPostfix
-	ExternalEntityLabelKeyKind       = "kind." + ExternalEntityLabelKeyPostfix
-	ExternalEntityLabelKeyVmName     = "name." + ExternalEntityLabelKeyPostfix
-	ExternalEntityLabelKeyTagPostfix = ".tag." + ExternalEntityLabelKeyPostfix
-	ExternalEntityLabelCloudVPCKey   = "vpc." + ExternalEntityLabelKeyPostfix
+	ExternalEntityLabelKeyPrefix       = "nephe.io/"
+	ExternalEntityLabelCloudPrefix     = "cloud-"
+	ExternalEntityLabelKeyTagPostfix   = "-tag"
+	ExternalEntityLabelKeyNamespace    = ExternalEntityLabelKeyPrefix + "namespace"
+	ExternalEntityLabelKeyKind         = ExternalEntityLabelKeyPrefix + "kind"
+	ExternalEntityLabelKeyVmName       = ExternalEntityLabelKeyPrefix + "vm-name"
+	ExternalEntityLabelVpcKey          = ExternalEntityLabelKeyPrefix + "vpc"
+	ExternalEntityLabelCloudRegionKey  = ExternalEntityLabelKeyPrefix + ExternalEntityLabelCloudPrefix + "region"
+	ExternalEntityLabelCloudVpcUIDKey  = ExternalEntityLabelKeyPrefix + ExternalEntityLabelCloudPrefix + "vpc-uid"
+	ExternalEntityLabelCloudVpcNameKey = ExternalEntityLabelKeyPrefix + ExternalEntityLabelCloudPrefix + "vpc-name"
+	ExternalEntityLabelCloudVmUIDKey   = ExternalEntityLabelKeyPrefix + ExternalEntityLabelCloudPrefix + "vm-uid"
+	ExternalEntityLabelCloudVmNameKey  = ExternalEntityLabelKeyPrefix + ExternalEntityLabelCloudPrefix + "vm-name"
 )
 
 const (
-	LabelCloudAccountName      = "cpa.name"
-	LabelCloudAccountNamespace = "cpa.namespace"
-	LabelCloudRegion           = "region"
-	LabelCloudVPCName          = "vpc.name"
+	// TODO: Prefix with VirtualMachine? or move to inventory package?
+	LabelPrefixNephe           = "nephe.io/"
```

Comment:
LabelPrefixNephe and ExternalEntityLabelKeyPrefix are same. Place make 1 const for it.

Reply:
Done